tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build/dist",
+    "module": "esnext",
+    "target": "es5",
+    "lib": ["es6", "dom"],
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "react",
+    "moduleResolution": "node",
+    "rootDir": "src",
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": false,
```

Comment:
Change this to `true`, and get it to compile :)

Reply:
![image](https://user-images.githubusercontent.com/9270263/47963491-bf2bb000-e035-11e8-9236-fddf28d437c2.png)

Reply:
https://media.giphy.com/media/rhfxbPtm4m5uo/giphy.gif